### PR TITLE
[15.0][ADD] event_registration_sms_compose

### PR DESCRIPTION
**Pull request by ivantodorovich**
Allow to send SMS text messages to multiple attendees at once, using the sms composer.

Same feature [proposed to odoo/master](https://github.com/odoo/odoo/pull/83801)

### CODE REVIEW

**Review by petrus-v — APPROVED** [review]

File: event_registration_sms_compose/views/event_registration.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="action_registration_sms_composer_single" model="ir.actions.act_window">
```
Comment (petrus-v) [inline event_registration_sms_compose/views/event_registration.xml]:
```suggestion
    <record id="action_registration_sms_composer_multi" model="ir.actions.act_window">
```

File: event_registration_sms_compose/views/event_registration.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                'default_composition_mode': 'mass',
+                'default_mass_keep_log': True,
+                'default_res_ids': active_ids
+            }
+        </field>
+        <field name="binding_model_id" ref="event.model_event_registration" />
+        <field name="binding_view_types">list</field>
+    </record>
+
+    <record id="action_registration_sms_composer_multi" model="ir.actions.act_window">
```
Comment (petrus-v) [inline event_registration_sms_compose/views/event_registration.xml]:
```suggestion
    <record id="action_registration_sms_composer_single" model="ir.actions.act_window">
```

**Review by petrus-v — COMMENTED** [review]
Small consistency xml_id labels !

**Comment by ivantodorovich** [comment]
> Small consistency xml_id labels !

Thanks! Done ;)

**Pull request closed without merging**